src/notification/notification.exceptions.ts

```diff
@@ -0,0 +1,7 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+
+export class EmailNotSent extends HttpException {
```

Comment:
I would rename this to EmailNotSentException.

Reply:
Fixed in #49